### ISSUE TITLE
Common Data Model and Extract, Transform & Load Tutorial

### ISSUE THREAD

**Issue by zohrehMehrabi** [imls-dmt/resources-workflow#446]
Information about registered data management training resource

- [ ] Date resource added: 3/2/2020
- [ ] Registered by: 
- [ ] Clearinghouse URL:https://dmtclearinghouse.esipfed.org/node/10656 

Tracking: @karlbenedict @njhoebel

**Comment by njhoebel** [comment]
This is a very good resource, Zohreh;  thanks for finding this.  I think it would be useful to add the other tutorials associated with this organization & topic too at:  
https://www.ohdsi.org/2019-tutorials-omop-common-data-model-and-standardized-vocabularies/ and https://www.ohdsi.org/past-events/2018-tutorials-omop-common-data-model-and-standardized-vocabularies/ 

unless they are substantially the same.  In that case, I would just add the 2019 tutorial.

The changes I made to the record include:
- Spelling out some of the acronyms in the description and adding a more complete list of what accompanies the video.  I also added the acknowledgment statement at the bottom for the funding and sponsoring organizations.
-  Added the funding organizations as contributors with a type of funding & sponsorship.
- Added a keyword for "community standards".
- Changed the Learning Resource Type to "learning activity" since it included spreadsheets, etc., that people could use with the slides.

One thing I would ask you to do is to change the order of the Author names to show as they do on the slides, i.e., with Rimma B first and Erica V last.  We try to put the authors in the names as they appear either on the slides or in the citation if there is one (not in this case, of course). The best way to do if the names are not already in the list of names is to enter the last person listed first and then go upwards on the list since Drupal puts the names in reverse order as they are entered.  Annoying, but there you go!  If this doesn't make sense, let me know.  Thanks!
